include/lsst/afw/cameraGeom/Detector.h

```diff
@@ -95,10 +94,33 @@ class Detector final {
                       TransformMap::Transforms const &transforms,
                       CrosstalkMatrix const &crosstalk = CrosstalkMatrix());
 
+    /**
+     * Make a Detector
+     *
+     * @param name name of detector's location in the camera
+     * @param id detector integer ID; used as keys in some tables
+     * @param type type of detector
+     * @param serial serial "number" that identifies the physical detector
+     * @param bbox bounding box
+     * @param ampInfoCatalog catalog of amplifier information
+     * @param orientation detector position and orientation in focal plane
+     * @param pixelSize pixel size (mm)
+     * @param transformMap coordinate systems and transforms between them
+     * @param crosstalk matrix of crosstalk coefficients
+     *
+     * @throws lsst::pex::exceptions::InvalidParameterError if: any amplifier
+     *     names are not unique
+     */
+    explicit Detector(std::string const &name, int id, DetectorType type, std::string const &serial,
+                      lsst::geom::Box2I const &bbox, lsst::afw::table::AmpInfoCatalog const &ampInfoCatalog,
+                      Orientation const &orientation, lsst::geom::Extent2D const &pixelSize,
+                      std::shared_ptr<TransformMap const> transformMap,
+                      CrosstalkMatrix const &crosstalk = CrosstalkMatrix());
+
     ~Detector() = default;
 
     Detector(Detector const &);
-    Detector(Detector &&);
+    Detector(Detector &&) = delete;
     Detector &operator=(Detector const &) = delete;
```

Comment:
Agreed; I had left this because I didn't want to break existing code, but now that I think about it a bit more, I don't think any existing code calling that could possibly be valid.